Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -293,7 +294,7 @@ struct PseudoStyleCache {
 };
 
 DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(RenderStyle);
-class RenderStyle {
+class RenderStyle : public CanMakeCheckedPtr<RenderStyle> {
```

Comment:
Is there a perf/memory use implication to doing this? RenderStyle is pretty much used everywhere across WebCore.